skia-safe/src/core/typeface.rs

```diff
@@ -155,22 +233,28 @@ impl RCHandle<SkTypeface> {
         }
     }
 
-    // TODO: implement this
-    pub fn may_support_kerning(&self) -> bool {
-        true
-    }
-
     // note: adjustments slice length must be equal to glyph's len - 1.
     pub fn kerning_pair_adjustments(&self, glyphs: &[GlyphId], adjustments: &mut [i32]) -> bool {
         (adjustments.len() == glyphs.len() + 1)
-        &&
-        unsafe {
-            self.native().
-                getKerningPairAdjustments(
+            && unsafe {
+                self.native().getKerningPairAdjustments(
                     glyphs.as_ptr(),
                     glyphs.len().try_into().unwrap(),
-                    adjustments.as_mut_ptr())
-        }
+                    adjustments.as_mut_ptr(),
+                )
+            }
+    }
+
+    pub fn new_family_name_iterator(&self) -> TypefaceLocalizedStrings {
+        TypefaceLocalizedStrings(unsafe { self.native().createFamilyNameIterator() })

```

Comment:
Check lifetime requirements on `TypefaceLocalizedStrings`!

Reply:
No lifetime requirement is stated in the Skia documentation. So I guess the iterator is copying the strings. I've added a testcase for that.